### PR TITLE
feat: Enhancements

### PR DESCRIPTION
**Pull request by ptbrowne**
Update deps
Use React

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,10 +38,14 @@
   "homepage": "https://github.com/cozy/cozy-interapp-sandbox#readme",
   "devDependencies": {
     "babel-preset-cozy-app": "^1.2.1",
+    "cozy-scripts": "^1.1.2",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
     "eslint": "5.9.0",
     "eslint-config-cozy-app": "1.1.4",
+    "eslint-config-prettier": "^3.3.0",
```
Comment (gregorylegarec) [inline package.json]: Shouldn't we be able to remove this line as `eslint-config-cozy-app` provides this dependency ? (I am not sure at all)

Comment (ptbrowne) [reply package.json]: No, unfortunately, yarn miss this dep :(

**Comment by nono** [comment]
@ptbrowne do you know why the travis build is red?

**Comment by CPatchane** [comment]
It's not recommended to use [`babel.config.js`](https://babeljs.io/docs/en/configuration) for babel 7 now. I pushed the fix to merge this PR

**Comment by ptbrowne** [comment]
Thanks for the reminder @nono :) 

Thanks also for the babel update @CPatchane . I see the build is still red, it seems jest has trouble loading JSX. Is it because the files have no .jsx extension ?

**Pull request closed without merging**